src/keas/kmi/facility.py

```diff
@@ -88,10 +146,44 @@ def decrypt(self, key, data):
             key=encryptionKey, mode=self.CipherMode,
             IV=self.initializationVector)
         # 3. Decrypt the data.
-        text = cipher.decrypt(data)
+        try:
+            text = cipher.decrypt(data)
+        except ValueError:
+            return data
+
         # 4. Remove padding and return result.
         return self._pkcs7Decode(text)
 
+    def decrypt_file(self, key, fsrc, fdst, chunksize=24*1024):
+        """ Decrypts a file using with the given key.
+        Parameters are similar to encrypt_file.
+        """
+        header = str(fsrc.read(2))
+        if header != '.e':
```

Comment:
After a discussion with @frisi we found out that the ".e" marker isn't needed in keas.kmi.

".e" is from https://github.com/zopefoundation/cipher.encryptingstorage and keas.kmi should just fail hard when it gets an unencrypted file on decrypt_file. 

So i'll remove the ".e" marker from encrypt_file and decrypt_file.

Thank you @strichter for pointing this out.